octobot_services/api/notification.py

```diff
@@ -44,7 +47,18 @@ async def send_notification(notification: Notification) -> None:
             }
         )
     except KeyError:
-        pass
+        if len(pending_notifications) < MAX_PENDING_NOTIFICATION:
+            pending_notifications.append(notification)
```

Comment:
Should we `pop` the first one and `append` the new one ?

Reply:
Not sure, we can but in this case it's better to increase MAX_PENDING_NOTIFICATION I guess, for now it's only used to store the "octobot is starting ..." message, you think there can be many more notifications waiting ?

Reply:
I don't think so. It can happen if the notifier is very long to start. I think the current MAX_PENDING_NOTIFICATION is enough :+1: